packages/next/next-server/server/next-server.ts

```diff
@@ -905,6 +905,13 @@ export default class Server {
       })
     }
 
+    const previewData = tryGetPreviewData(
```

Comment:
Let's skip this check unless the page is SSG or GSSP (as it parses cookies).

Reply:
Updated